### ISSUE TITLE
ASNI cursor movement control is not supported

### ISSUE THREAD

**Issue by JakeWharton** [katharostech/cast2gif#10]
Listed at https://en.wikipedia.org/wiki/ANSI_escape_code#Terminal_output_sequences

I have a CLI which moves the cursor to repaint existing content, but the GIF always appends content ignoring these control codes. Color codes do seem to be supported, however.

**Comment by zicklag** [comment]
`cast2gif` doesn't handle the terminal emulation itself, so it might be a problem with the upstream terminal emulator [`vt100`](https://github.com/doy/vt100-rust). I'd have to test it to be sure. Could you upload a zip of the cast file ( or put it in a GitHub [Gist](https://gist.github.com) ) so I can test it out?

**Comment by JakeWharton** [comment]
Yep! Here's the JSON, the SVG output (from svg-term), and the GIF output from cast2gif: https://gist.github.com/JakeWharton/0b2a7233ea513250b7a21a3070d350ba.
